### ISSUE TITLE
Modes for 5:4 aspect ratio displays

### ISSUE THREAD

**Issue by shift838** [ramapcsx2/gbs-control#189]
I would like to see resolutions for the standard 5:4 aspect ratio monitors.  not all of us want a widescreen display and would like to use this on resolutions like 1024x768 for some older computers like the Geneve 9640

**Comment by LackofTrack** [comment]
Isn't 1280x1024 a 5:4 resolution?

**Comment by shift838** [comment]
Yes, but the monitor I have has a resolution of 1024 x 768 max.

**Comment by t9999clint** [comment]
I have a few displays that can do 1600x1200
It'd be pretty wicked to have that as a option. It also allows for easy integer scaling for a bunch of different resolutions

**Comment by shift838** [comment]
I agree that more than one 5:4 aspect ratio should be offered.  This would expand the options for gbscontrol even further for users.

**Comment by zajc3w** [comment]
second that, just spend a day modifying 1280x1024 to 1024x768@60HZ
It's a bit of a bodge, most parameters didn't make sense to me but it works, at least with 50hz rgbhv source
Try moving screen in web GUI if it appears chopped up after loading.


[pal_1280x1024.zip](https://github.com/ramapcsx2/gbs-control/files/6507373/pal_1280x1024.zip)

**Comment by ramapcsx2** [comment]
If you spend 2-3 days more, you have a good shot at this! :)

**Comment by mpalpha** [comment]
https://github.com/mpalpha/gbs-control-odv/releases/tag/1.0.24
